testsuite/MathMLToDisplay/Characters/issue898.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <title>SVG output of space characters</title>
+    <!-- Copyright (c) 2015 MathJax Consortium
+         License: Apache License 2.0 -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <script type="text/javascript" src="../../header.js"></script>
+    <script type="text/javascript" src="../../scriptTests.js"></script>
+    <script type="text/javascript">
+      function getTestCases() {
+        var texts = document.getElementsByTagName('text');
+        var check1 = (texts.length === 0) ;        
+        var svgs = document.getElementsByTagName('svg');
+        var check2 = true;
+        for (i=0;i<svgs.length;i++){
```

Comment:
The convention we've been using is to use spaces around binary operators and relations, and after semicolons, so this would be

    for (i = 0; i < svgs.length; i++) {

Technically, it should be

    for (var i = 0; i < svgs.length; i++) {

to localize the variable `i`.  Though it doesn't cause problems here, it is good to be in the practice of keeping all your local variables local.